client/components/domains/domain-suggestion/style.scss

```diff
@@ -47,6 +47,11 @@
 		.is-placeholder & {
 			color: transparent;
 		}
+
+		.flag {
```

Comment:
@mtias It made sense to me to put it in `h3` because that's where it lives. It would work outside too. Was trying to make it most-specific.